src/widget/wlibrarysidebar.cpp

```diff
@@ -183,15 +183,16 @@ bool WLibrarySidebar::isLeafNodeSelected() {
 }
 
 void WLibrarySidebar::keyPressEvent(QKeyEvent* event) {
-    if (event->key() == Qt::Key_Return) {
+    const int key = event->key();
+    if (key == Qt::Key_Return) {
```

Comment:
I think this could be a switch statement with a `default:` bit at the bottom.

Reply:
Haha, I thought about that earlier but wanted to keep the changes minimal.
Now, without the `view()->isVisible()` branch it makes perfect sense and is clean, too.

Done.

Reply:
ping. this is still an else if construct, you only changed wsearchlineedit and not wlibrarysidebar.

Reply:
oh, overlooked the file name.
Done.